cwl/src/main/scala/cwl/MyriadInputTypeToSortedCommandParts.scala

```diff
@@ -72,18 +88,20 @@ object MyriadInputTypeToSortedCommandParts extends Poly1 {
 
 object MyriadInputInnerTypeToSortedCommandParts extends Poly1 {
 
-  import Case._
+  type CommandPartBuilder = (Option[InputCommandLineBinding], WomValue, CommandBindingSortingKey, Boolean, ExpressionLib, SchemaDefRequirement) => Option[CommandPartsList]
 
-  def ex(component: String) = throw new RuntimeException(s"input type $component not yet supported by WOM!")
+  import Case._
 
   // Primitive type: we just need to create a command part from the binding if there's one here.
   implicit def ct: Aux[CwlType, CommandPartBuilder] = {
     at {
       cwlType => {
-        (inputBinding, womValue, key, hasShellCommandRequirement, expressionLib) => {
-          cwlType match {
-            case CwlType.Null => CommandPartsList.empty
-            case _ => inputBinding.toList.map(_.toCommandPart(key, womValue, hasShellCommandRequirement, expressionLib))
+        case (_, WomOptionalValue(_, None), _, _, _, _) => CommandPartsList.empty.some
```

Comment:
```
scala> import cats.syntax.option._
import cats.syntax.option._

scala> val s: String = null
s: String = null

scala> s.some
res0: Option[String] = Some(null)

scala> Option(s)
res1: Option[String] = None
```

😞

Reply:
I get your point, but this instance is guaranteed non-null.  I'm working on a `.option` PR for cats for general usage.

Reply:
https://github.com/typelevel/cats/pull/2235